### ISSUE TITLE
Fleeing mice can be bugged into breaking their AI and spamming runtimes every 0.1 seconds (each mouse)

### ISSUE THREAD

**Issue by iprice** [tgstation/tgstation#72166]
Reporting client version: 514.1589

## Round ID:
[196763](https://scrubby.melonmesa.com/round/196763)

## Testmerges:
- [holopara qol update](https://www.github.com/tgstation/tgstation/pull/72042)
- [[READY] DRAMATIC SHUTTLES!! You can now fly around the shuttle](https://www.github.com/tgstation/tgstation/pull/71906)

## Reproduction:

Easiest in a supply pod launch facility room, not sure exactly what breaks the pathing but once its broken it stays broken

In a supply pod launch facility, spawn in a brown mouse (spawn mouse/brown) and place a felinid next to them
![image](https://user-images.githubusercontent.com/3789327/209065665-900b1829-3909-4047-8cb0-2e2c564fc84c.png)
The mouse will flee into the corner of the pod room and hump the wall squeaking as it does.
Arrange a supply pod, reverse drop mode, to place the mob into the same facility (for some reason just getting mobs around doesn't work, maybe something to do with unable to path /inside/ the pod?) e.g.
![image](https://user-images.githubusercontent.com/3789327/209065811-bb75952b-3b6e-4c74-9ad5-ff0600361b56.png)
Scoop the mouse from the corner and have it drop next to the felinid again.

The mouse will no longer move.  At all.  and it generates runtimes every 1/10th of a second

```
[2022-12-22 05:42:11.067] runtime error: The mouse wants to perform action type /datum/ai_behavior/run_away_from_target/mouse which requires movement, but has no current movement target! (code/datums/ai/_ai_controller.dm:170)
 - proc name:  stack trace (/proc/_stack_trace)
 -   source file: stack_trace.dm,4
 -   usr: (src)
 -   src: null
 -   call stack:
 -  stack trace("The mouse wants to perform act...", "code/datums/ai/_ai_controller....", 170)
 - /datum/ai_controller/basic_con... (/datum/ai_controller/basic_controller/mouse): process(0.1)
 - AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): fire(0)
 - AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): fire(0)
 - AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): ignite(0)
 - Master (/datum/controller/master): RunQueue()
 - Master (/datum/controller/master): Loop(2)
 - Master (/datum/controller/master): StartProcessing(0)
 - 
[2022-12-22 05:42:11.172] runtime error: The mouse wants to perform action type /datum/ai_behavior/run_away_from_target/mouse which requires movement, but has no current movement target! (code/datums/ai/_ai_controller.dm:170)
 - proc name:  stack trace (/proc/_stack_trace)
 -   source file: stack_trace.dm,4
 -   usr: (src)
 -   src: null
 -   call stack:
 -  stack trace("The mouse wants to perform act...", "code/datums/ai/_ai_controller....", 170)
 - /datum/ai_controller/basic_con... (/datum/ai_controller/basic_controller/mouse): process(0.1)
 - AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): fire(0)
 - AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): fire(0)
 - AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): ignite(0)
 - Master (/datum/controller/master): RunQueue()
 - Master (/datum/controller/master): Loop(2)
 - Master (/datum/controller/master): StartProcessing(0)
 - 
[2022-12-22 05:42:11.266] runtime error: The mouse wants to perform action type /datum/ai_behavior/run_away_from_target/mouse which requires movement, but has no current movement target! (code/datums/ai/_ai_controller.dm:170)
 - proc name:  stack trace (/proc/_stack_trace)
 -   source file: stack_trace.dm,4
 -   usr: (src)
 -   src: null
 -   call stack:
 -  stack trace("The mouse wants to perform act...", "code/datums/ai/_ai_controller....", 170)
 - /datum/ai_controller/basic_con... (/datum/ai_controller/basic_controller/mouse): process(0.1)
 - AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): fire(0)
 - AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): fire(0)
 - AI Behavior Ticker (/datum/controller/subsystem/processing/ai_behaviors): ignite(0)
 - Master (/datum/controller/master): RunQueue()
 - Master (/datum/controller/master): Loop(2)
 - Master (/datum/controller/master): StartProcessing(0)
 - 
```

**Comment by Jacquerel** [comment]
Hopefully was fixed by https://github.com/tgstation/tgstation/pull/72130

**Comment by iprice** [comment]
> Hopefully was fixed by #72130, at the very least I can no longer reproduce it on master but I might have been following the steps incorrectly.

Does seem fixed, at least the mouse no longer "soft locks" in my reproduction.  Thanks.
